concord-server/server/src/main/java/com/rabidgremlin/concord/api/SystemStats.java

```diff
@@ -0,0 +1,134 @@
+package com.rabidgremlin.concord.api;
+
+import java.util.List;
+
+import com.google.common.base.Objects;
+
+public class SystemStats
+{
+
+  private final int totalPhrases;
+
+  private final int completedPhrases;
+
+  private final int phrasesWithConsensus;
+
+  private final int phrasesWithConsensusNotCompleted;
+
+  private final int labelsUsed;
+
+  private final int totalVotes;
+
+  private final int totalLabels;
+
+  private final int userCount;
+
+  private final List<LabelCountStats> labelCountStats;
+
+  public SystemStats(int totalPhrases, int completedPhrases, int phrasesWithConsensus, int phrasesWithConsensusNotCompleted, int labelsUsed, int totalVotes,
+    int totalLabels,
+    int userCount, List<LabelCountStats> labelCountStats)
+  {
+    this.totalPhrases = totalPhrases;
+    this.completedPhrases = completedPhrases;
+    this.phrasesWithConsensus = phrasesWithConsensus;
+    this.phrasesWithConsensusNotCompleted = phrasesWithConsensusNotCompleted;
+    this.labelsUsed = labelsUsed;
+    this.totalVotes = totalVotes;
+    this.totalLabels = totalLabels;
+    this.userCount = userCount;
+    this.labelCountStats = labelCountStats;
+  }
+
+  public int getTotalPhrases()
+  {
+    return totalPhrases;
+  }
+
+  public int getCompletedPhrases()
+  {
+    return completedPhrases;
+  }
+
+  public int getPhrasesWithConsensus()
+  {
+    return phrasesWithConsensus;
+  }
+
+  public int getPhrasesWithConsensusNotCompleted()
+  {
+    return phrasesWithConsensusNotCompleted;
+  }
+
+  public int getLabelsUsed()
+  {
+    return labelsUsed;
+  }
+
+  public int getTotalVotes()
+  {
+    return totalVotes;
+  }
+
+  public int getTotalLabels()
+  {
+    return totalLabels;
+  }
+
+  public int getUserCount()
+  {
+    return userCount;
+  }
+
+  public List<LabelCountStats> getLabelCountStats()
+  {
+    return labelCountStats;
+  }
+
+  @Override
+  public boolean equals(Object o)
```

Comment:
I can't see where you use this logic?

Reply:
in the unit tests, `assertThat` uses it under the hood